pkg/containerd/utility_windows.go

```diff
@@ -0,0 +1,15 @@
+// +build windows
+
+package containerd
+
+import (
+	"fmt"
+)
+
+func OverlaySupported(root string) error {
+	return fmt.Errorf("%s is not supported on Windows", root)
```

Comment:
Can we define a standard ErrUnsupportedPlatform or the like (outside this package), and and observe a convention of returning or `errors.Wrap`ing that for things that are not supported on Windows?

Reply:
@brandond sure, just let me know which approach you want and I will do my best to implement it.

Reply:
@brandond I just pushed the changes to make that correction.